src/Illuminate/Database/Query/Builder.php

```diff
@@ -844,6 +844,29 @@ public function orWhere($column, $operator = null, $value = null)
         return $this->where($column, $operator, $value, 'or');
     }
 
+    /**
+     * Add an "when where" clause that runs "where" when condition is true.
+     *
+     * @param  mixed  $condition
+     * @param  \Closure|array|string|\Illuminate\Database\Query\Expression  $column
+     * @param  mixed  $operator
+     * @param  mixed  $value
+     * @param  string  $boolean
```

Comment:
param type and the variable do not match. string => Boolean ? And formatting seems off.

Reply:
@TNChalise it's the same as in method where() docblock. `$boolean` here means values 'and' or 'or'.